src/index.ts

```diff
@@ -134,6 +147,35 @@ async function getTestFiles(execute: ExecuteWithScope) {
     };
 }
 
+/**
+ * Finds all the test files that match a given list of strings. If the list is empty,
+ * it finds all *.test.brs files.
+ * @param fileMatches A list of file path matches from the command line
+ */
+async function globMatchFiles(fileMatches: string[]) {
+    fileMatches = fileMatches.map((match) => {
+        if (path.parse(match).ext === ".brs") {
+            // If the string is a brs file, use it directly.
+            return match;
+        } else {
+            // If the string is not already a brs file, do partial matches on brs files
+            // that contain the string, and also treat the string as a directory.
+            return `{*${match}*.brs,*${match}*/**/*.brs}`;
```

Comment:
`*` for days, i know...

in testing my logic, this seems to work as expected. but i was also kind of unsure how to write unit and/or e2e tests for this.